src/main/java/com/signavio/examples/dmn/testsuite/TestResult.java

```diff
@@ -24,8 +34,11 @@ public static TestResult failure(TestCase testCase) {
 	}
 	
 	
-	@Override
-	public String toString() {
-		return String.valueOf(success) + " -> " + testCase.toString();
+	public abstract static class TestCaseMixin {
+		
+		@JsonIgnore
+		public abstract List<DefaultParameter> getAllExpectedParametersInternal();
```

Comment:
what is this needed for?

Reply:
If you just convert a testcase into json, jackson picks up all getters into fields and this getter is something that i wanted to hide.